tests/test_funcs.py

```diff
@@ -21,10 +21,24 @@
 from array_api_extra._lib._testing import xp_assert_close, xp_assert_equal
 from array_api_extra._lib._utils._compat import device as get_device
 from array_api_extra._lib._utils._typing import Array, Device
+from array_api_extra.testing import lazy_xp_function
 
 # some xp backends are untyped
 # mypy: disable-error-code=no-untyped-usage
 
+lazy_xp_function(atleast_nd, static_argnames=("ndim", "xp"))
+lazy_xp_function(cov, static_argnames="xp")
+# FIXME .device attribute https://github.com/data-apis/array-api-compat/pull/238
```

Comment:
https://github.com/data-apis/array-api-compat/pull/238

Evidence that the numpy-compat PR works: https://github.com/crusaderky/array-api-extra/commit/d27f75d385362d3312fba7f2f2bb1338e9d1f502